src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -300,11 +300,6 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_cancel_recv(MPIR_Request * rreq)
   fn_exit:
     MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPIDI_NM_MPI_CANCEL_RECV);
     return mpi_errno;
-
-#ifndef MPIDI_BUILD_CH4_SHM
-  fn_fail:
-    goto fn_exit;
-#endif
```

Comment:
Oops, just realized that this would fix an issue with one of our internal patches which hasn't been pushed here yet. I'll close the PR for now.